### DISCUSSION TITLE
Can't scan barcode with LORIS, client-side issue

### DISCUSSION THREAD

**Discussion by cmion1** [aces/Loris#8214] [Q&A] [answered]
Hello
I work as Technical Support in an university using the app, they are having difficulties scanning barcodes with iPhones, iPad, etc…
Is there any troubleshooting steps to take particular to the app? Do browsers needs special settings?

**Comment by LeighMac** [comment, marked as answer]
Hi Christophe, 

If this is regarding HBCD, can you contact the site monitors for HBCD, they should be able to assist with the particular SOPs and protocols (and escalate if necessary). 

If this is not regarding HBCD, can you let us know what project you are referring to?
